.idea/gradle.xml

```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
```

Comment:
why have you removed this all?
You don't need to send this file in pull request.